### DISCUSSION TITLE
v0.12.8 - The V-day 2022 Update

### DISCUSSION THREAD

**Discussion by ThePotatoGuy** [Monika-After-Story/MonikaModDev#8682] [Announcements & Releases]
_I'm sorry I can't come out of this screen :(_

# Gameplay Changes

* F14 content

* added necklace selector

* 5-year anniversay updated

# Topic Changes

### Additions
* 1 new story

# Fixes

* typofixes

* fixed "familiar-" and "skill" from being considered bad names

# API Changes

* `MASImageTagDecoDefinition.register_img_same` now raises an exception on init if it could not find a valid deco definition from source

## What's Changed
* Add test topic and exception for `register_img_same` by @ThePotatoGuy in https://github.com/Monika-After-Story/MonikaModDev/pull/8393
* [Story] - "Mind the Gap" by @lunulae in https://github.com/Monika-After-Story/MonikaModDev/pull/8606
* [Rework] - Anniversary 5 by @lunulae in https://github.com/Monika-After-Story/MonikaModDev/pull/8572
* Bad name false positive fix by @dreamscached in https://github.com/Monika-After-Story/MonikaModDev/pull/8656
* [F14] - "Happy Valentine's Day, Monika!" by @lunulae in https://github.com/Monika-After-Story/MonikaModDev/pull/8632
* f14 2022 by @jmwall24 in https://github.com/Monika-After-Story/MonikaModDev/pull/8648
* 0.12.7 Typos by @multimokia in https://github.com/Monika-After-Story/MonikaModDev/pull/8594

**Full Changelog**: https://github.com/Monika-After-Story/MonikaModDev/compare/v0.12.7...v0.12.8

<hr /><em>This discussion was created from the release <a href='https://github.com/Monika-After-Story/MonikaModDev/releases/tag/v0.12.8'>v0.12.8 - The V-day 2022 Update</a>.</em>

**Comment by dennyrodrigo1** [comment]
After the update was done I opened the game and all of the gifts that I have given to Monika and her hairstyles are gone. So I tried giving her the gifts back again and she doesn't receive them. Is there a new way to gift her something?

**Reply by jmwall24** [reply]
> After the update was done I opened the game and all of the gifts that I have given to Monika and her hairstyles are gone. So I tried giving her the gifts back again and she doesn't receive them. Is there a new way to gift her something?

How did you update the mod? Sounds like you may have replaced instead of merged? Reinstall the spritepacks (you won't need to regift) and see if that helps.

**Reply by dennyrodrigo1** [reply]
Once I've downloaded the sprite pack what do I do with it? Sorry if I'm being too much with this.

**Reply by jmwall24** [reply]
Watch this video on spritepack installation. For anything you've already gifted, you can skip the gift step. You would do this for each spritepack. If you downloaded the combined version, it's the same thing, you'd just only do it once. https://www.youtube.com/watch?v=bjJ0U8l1m8c

**Reply by Feigi53** [reply]
so after I installed the update when I try to play your reality it on the piano doesn't work she'll act as if I've screwed up when I'm getting all the notes right and keeps restarting me at the beginning

**Reply by jmwall24** [reply]
@Feigi53 there's been a recent change to the piano notes due to an error we had before, refer to `game\mod_assets\games\piano\songs\yourreality.json` for the correct notes. Can be opened with any text editor.

**Reply by Feigi53** [reply]
Thank you

**Comment by SpectraPhantom99** [comment]
will there be more updates?

**Reply by jmwall24** [reply]
Yes

**Reply by Booplicate** [reply]
Are you sure?

**Reply by jmwall24** [reply]
Yes

**Comment by DABnoREGRET** [comment]
Sorry to ask but what's the difference between the normal and dlx version? I can't find any info regarding this.

**Reply by multimokia** [reply]
Deluxe (dlx) comes with all spritepacks pre-installed

**Comment by PAPIscrook** [comment]
anyone who may understand, while installing the update, i got a error OSERROR 13 permission denied
if there any possible way for me to fix this. if its like the og game, what files should i save before deleting and reinstalling with the latest update?
honestly any help is appreciated. thatnk you and have a great day
